app/src/main/java/com/github/multimatum_team/multimatum/model/DeadlineAdapter.kt

```diff
@@ -0,0 +1,88 @@
+package com.github.multimatum_team.multimatum.model
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Color
+import android.graphics.Typeface
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.BaseAdapter
+import android.widget.TextView
+import com.github.multimatum_team.multimatum.R
+/**
+Class who is used to show the deadline in a clear list.
```

Comment:
Nitpick: the docstring should be indented and attached to the class declaration.